.github/workflows/ci.yml

```
@@ -20,9 +20,20 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - name: Install native dependencies
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
-      - run: cargo check --workspace 
+      - run: cargo check --workspace
+      - run: cargo check --features=hot_reload,buffer_labels
```

I *think* this should already be checked by `--workspace`, as that implies a unified set of features.

However, I'm not sure that the inverse is being checked (base `vello` with no features). That's unfortunate, as that is the state actual users should be accessing.
I'm not sure how to properly test that

---

I don't believe `--workspace` covers features in any way. Also, we're not setting default features, so the run without features set should cover that.

---

My reason for suspecting it would check all features is that `with_winit` requires `buffer_labels` and `hot_reload`.

However, you are correct that a "root" `cargo check` only checks Vello without features, so I believe this should be:
```suggestion
      # Check vello (the default crate) without the features used by `with_winit` for debugging
      - run: cargo check 
```